cli/src/main/java/com/devonfw/tools/ide/tool/node/Node.java

```diff
@@ -23,6 +25,16 @@ public Node(IdeContext context) {
     super(context, "node", Set.of(Tag.JAVA_SCRIPT, Tag.RUNTIME));
   }
 
+  @Override
+  protected void postInstall() {
+    super.postInstall();
+    Npm npm = this.context.getCommandletManager().getCommandlet(Npm.class);
+    ProcessResult result = npm.runPackageManager("config", "set", "prefix", getToolPath().toString());
```

Comment:
could this cause a cyclic dependency since `npm` depends on `node` and here we depend from `node` on `npm`?

Reply:
OK. Let's be optimistic and try it out.